tests/MongoDB.Driver.Core.TestHelpers/XunitExtensions/RequirePlatform.cs

```diff
@@ -78,7 +78,7 @@ public RequirePlatform SkipWhen(SupportedOperatingSystem operatingSystem, params
         {
             var currentOperatingSystem = GetCurrentOperatingSystem();
             var currentTargetFramework = GetCurrentTargetFramework();
-            if (operatingSystem == currentOperatingSystem && (targetFrameworks == null || targetFrameworks.Contains(currentTargetFramework)))
+            if (operatingSystem == currentOperatingSystem && ((targetFrameworks?.Length ?? 0) == 0 || targetFrameworks.Contains(currentTargetFramework)))
```

Comment:
a minor test fix. Without this fix, tests that don't configure target frameworks explicitly won't be skipped